.vib/redis/goss/goss.yaml

```diff
@@ -0,0 +1,64 @@
+{{- $auth := printf "REDISCLI_AUTH=%s" .Vars.auth.password }}
+{{- $master_endpoint := printf "-h redis-master -p %d" .Vars.master.service.ports.redis }}
+{{- $replicas_endpoint := printf "-h redis-replicas -p %d" .Vars.replica.service.ports.redis }}
+{{- $replicas := .Vars.replica.replicaCount }}
+command:
+  {{- $key := printf "key_%s" (randAlpha 5) }}
+  {{- $value := printf "value_%s" (randAlpha 5) }}
+  redis-set-key-value-pairs:
+    exec: export {{ $auth }} && redis-cli {{ $master_endpoint }} SET {{ $key }} {{ $value }} {{ range $e, $i := until $replicas }} && redis-cli -h redis-replicas-{{ $i }}.redis-headless -p {{ $.Vars.replica.containerPorts.redis }} GET {{ $key }} | grep -q {{ $value }}{{ end }}
+    exit-status: 0
+  redis-disabled-command:
+    exec: {{ $auth }} redis-cli {{ $master_endpoint }} {{ .Vars.master.disableCommands }}
+    exit-status: 0
+    stdout:
+      - ERR unknown command
+  redis-master-role:
+    exec: {{ $auth }} redis-cli {{ $master_endpoint }} ROLE
+    exit-status: 0
+    stdout:
+      - master
+      {{ range $e, $i := until $replicas }}
+      - redis-replicas-{{ $i }}
+      {{ end }}
+  redis-replicas-role:
+    exec: {{ $auth }} redis-cli {{ $replicas_endpoint }} ROLE
+    exit-status: 0
+    stdout:
+      - slave
+  check-user-info:
+    exec: id
+    exit-status: 0
+    stdout:
+    - uid={{ .Vars.master.containerSecurityContext.runAsUser }}
+    - /groups=.*{{ .Vars.master.podSecurityContext.fsGroup }}/
+file:
+  {{ .Vars.master.persistence.path }}:
+    filetype: directory
+    exists: true
+    mode: "2775"
+    owner: root
+  /opt/bitnami/scripts/start-scripts:
+    filetype: directory
+    exists: true
+    mode: "2777"
+    owner: root
```

Comment:
I would omit this path. There already are a bunch of other paths in the tests, but in particular, this one is used by the primary container to start the Redis service. An error related to this mounted volume would probably end up as a deployment failure.

Reply:
You are right, removed both of them.